css/styles.css

```diff
@@ -1112,7 +1114,9 @@
   /* Contact form section */
   #foot {
     box-sizing: border-box;
-    background: url("../images/bg-contact.png");
+    /* background: url("../images/bg-contact.png"); */
```

Comment:
- [ OPTIONAL ] Here as well, kindly remove the commented-out code 👍🏽 because it makes your code messy 🌪️ You can always check your old commits for snippets of codes you have already committed in the previous milestone 😁